app/src/main/kotlin/com/simplemobiletools/keyboard/activities/MainActivity.kt

```diff
@@ -12,28 +12,32 @@ import com.simplemobiletools.commons.helpers.LICENSE_GSON
 import com.simplemobiletools.commons.models.FAQItem
 import com.simplemobiletools.keyboard.BuildConfig
 import com.simplemobiletools.keyboard.R
-import kotlinx.android.synthetic.main.activity_main.*
+import com.simplemobiletools.keyboard.databinding.ActivityMainBinding
 
 class MainActivity : SimpleActivity() {
+    private val binding by lazy(LazyThreadSafetyMode.NONE) { ActivityMainBinding.inflate(layoutInflater) }
```

Comment:
lets actually use `viewBinding` like in Dialer, that is the helper extension function

Reply:
done